components/contributors/contributors.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit, Input, Inject } from '@angular/core';
+import { RouterLink } from '@angular/router-deprecated';
+import { Angulartics2On } from 'angulartics2/index';
+import * as _ from 'lodash';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/from';
+import 'rxjs/add/operator/mergeMap';
+import 'rxjs/add/operator/toArray';
+import { AsyncPipe } from '@angular/common';
+import { ContenfulContent } from '../contentfulService/contentful-content.service';
+import { ContentfulProfilePage, ContentfulContributionPage } from '../contentfulService/aliases.structures';
+
+@Component({
+  selector: 'gm-contributors',
+  template: require('./contributors.html') as string,
+  directives: [RouterLink, Angulartics2On],
+  styles: [require('./contributors.css') as string],
+  pipes: [AsyncPipe]
+})
+
+export class ContributorsComponent implements OnInit {
+  @Input() private idNodePage: string;
+  private profiles: ContentfulProfilePage;
+  private contentfulContentService: ContenfulContent;
+  private hide: boolean = false;
+
+  public constructor(@Inject(ContenfulContent) contentfulContentService: ContenfulContent) {
+    this.contentfulContentService = contentfulContentService;
+  }
+
+  public ngOnInit(): any {
+
+    this.contentfulContentService
+      .getContributionsOf(this.idNodePage)
+      .mergeMap((contributions: any) => Observable.from(contributions))
+      .mergeMap((contrubution: ContentfulProfilePage) => this.contentfulContentService.getProfilesOf(contrubution.sys.id))
```

Comment:
batch load of profiles 
https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters/array-with-multiple-values/query-entries